src/schema/resolvers.ts

```diff
@@ -23,6 +23,43 @@ export const resolvers = {
       }
       return user;
     },
+
+    async users(_parent, args: { data: { limit: number; page: number } }, context: { token: string }) {
+      new Authenticator().isTokenValid(context.token);
+      const page = args.data.page ?? 1;
+      const take = args.data.limit ?? 10;
+      const skip = take * (page - 1);
+      if (page < 0) {
+        throw new CustomError(`page must not be negative`, 400);
+      }
+      if (take < 0) {
+        throw new CustomError(`limit must not be negative`, 400);
+      }
+
+      const userRepository = getRepository(User);
+      const [users, count] = await userRepository.findAndCount({ order: { name: 'ASC' }, skip, take });
+
+      const totalPage = Math.floor(count / take);
+      const hasPreviousPage = page > 1 ? true : false;
+      const hasNextPage = page < totalPage ? true : false;
```

Comment:
```suggestion
      const hasPreviousPage = page > 1;
      const hasNextPage = page < totalPage;
```

Lembrando que comparações já são expressões booleanas, então não há necessidade de ternário.